.github/workflows/docs-quality.yml

```
@@ -38,3 +44,170 @@ jobs:
+          adr_index = adr_index_path.read_text(encoding='utf-8')
+          adr_old = '| [0006](0006-vram-gpu-nvidia-orchestration.md) | VRAM-adaptive GPU compute and model-credential boundary | Accepted | accepted-target | LLM orchestration policy superseded by ADR 0010; autonomous development authority governed by ADR 0015. |'
+          adr_new = '| [0006](0006-vram-gpu-nvidia-orchestration.md) | VRAM-adaptive GPU compute and model-credential boundary | Accepted | partial | VRAM budgets, executable OOM retries, and compensated CPU `f64` reference are on the active PR; fixed-pool CPU multithreading, live GPU kernels, mixed-precision device lanes, and hardware parity remain accepted-target. |'
+          if adr_index.count(adr_old) != 1:
+              raise SystemExit('ADR index compute target mismatch')
+          adr_index_path.write_text(adr_index.replace(adr_old, adr_new, 1), encoding='utf-8')
+
+          standards_path = Path('docs/research/standards-and-literature.md')
+          standards = standards_path.read_text(encoding='utf-8')
+          section = '''## Numerical backends, VRAM, and mixed precision
+
+IEEE. (2019). *IEEE standard for floating-point arithmetic* (IEEE Std 754-2019). https://doi.org/10.1109/IEEESTD.2019.8766229
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)